src/common/translation/i18n/fi.json

```
@@ -557,16 +551,16 @@
         }
       },
       "child": {
-        "birthdate": {
+        "birthyear": {
           "input": {
-            "label": "Lapsen syntymäaika (pp.kk.vvvv)",
-            "placeholder": "Lisää lapsen syntymäaika"
+            "label": "Lapsen syntymäaika (vvvv)",
```

```suggestion
            "label": "Lapsen syntymävuosi",
```
i.e. "Child's birthyear" in Finnish